MacawTests/MacawSVGTests.swift

```diff
@@ -7,139 +7,15 @@ import XCTest
 #endif
 
 class MacawSVGTests: XCTestCase {
-    
-    /*
-     When test are running, if shouldSaveFaildedTestImage set to true, result images will be saved into MacawTestOutputData folder in documents.
-     
-     Also, there is no way to detect that multiple test will runs.
-     In this case, when all MacawSVGTests will be performed, set multipleTestsWillRun to true, then all test images will be saved to the folder.
-     
-     Then, if you want to investigate one particular test result, set multipleTestsWillRun to false and test folder will be deleted before new test will run.
-     */
-    
-    private let testFolderName = "MacawTestOutputData"
-    private let shouldComparePNGImages = true
-    private let multipleTestsWillRun = false
-    private let shouldSaveFaildedTestImage = false
```

Comment:
You have removed few options in tests. For example, there is no `shouldSaveFaildedTestImage` which means that tests can generate some data which is not a good default value.

I would recommend to remove all these changes from this PR to keep it only about pattern support and create another one especially for tests improvements where we can discuss these changes.